core/src/data/dataSource.h

```diff
@@ -105,6 +115,12 @@ class DataSource : public std::enable_shared_from_this<DataSource> {
     // Name used to identify this source in the style sheet
     std::string m_name;
 
+    // Minimum zoom for which tiles will be displayed
+    int32_t m_minDisplayZoom;
```

Comment:
I can switch `m_minDisplayZoom` to be `int16_t`, however, that would be inconsistent with m_maxZoom which has always been `int32_t`. We really could have all these zoom vars be `int8_t` (max value of 127)